lxd/db/cluster/secrets.go

```diff
@@ -301,3 +301,64 @@ func deleteSecretsByID(ctx context.Context, tx *sql.Tx, ids ...int) error {
 
 	return nil
 }
+
+// GetAllBearerIdentitySigningKeys returns a map of identity ID to token signing keys. It should only be used to refresh
```

Comment:
Just a nit, no need to change, but comments like this look pretty odd to me, why not something like this?

```
// GetAllBearerIdentitySigningKeys returns a map of identity ID to token signing keys. 
// It should only be used to refresh the identity cache.
```

Reply:
Yeah I tend to try and break things up by sentence by default, but also not have an unlimited line length to enable effective review in Github. So I would say use your judgement.